aws/templates/solution/solution_rds.ftl

```diff
@@ -1,164 +1,192 @@
 [#-- RDS --]
-[#if (componentType == "rds") && deploymentSubsetRequired("rds", true)]
+[#if (componentType == "rds") ]
 
     [#assign db = component.RDS]
     
-    [#assign engine = db.Engine]
-    [#switch engine]
-        [#case "mysql"]
-            [#assign engineVersion =
-                db.EngineVersion?has_content?then(
-                    db.EngineVersion,
-                    "5.6"
-                )
-            ]
-            [#assign family = "mysql" + engineVersion]
-        [#break]
-
-        [#case "postgres"]
-            [#assign engineVersion =
-                db.EngineVersion?has_content?then(
-                    db.EngineVersion,
-                    "9.4"
-                )
-            ]
-            [#assign family = "postgres" + engineVersion]
+    [#list getOccurrences(component, deploymentUnit) as occurrence]
+        [#assign engine = occurrence.Engine]
+        [#switch engine]
```

Comment:
Suggest adding default for port if no content provided
Also suggest default size per engine type